tests/functional/replication/queueProcessor.js

```diff
@@ -81,6 +81,7 @@ class S3Mock {
                         size: constants.body.length,
                         start: 0,
                         dataStoreName: 'file',
+                        dataStoreETag: `1:${this.contentMd5}`,
```

Comment:
just make sure we're compatible with old data as well (if they don't have this field in metadata stored prior to the introduction of `dataStoreETag`)